Modules/TemplateManager.cs

```diff
@@ -119,7 +134,11 @@ public static void SendTemplate(string str = "", byte playerId = 0xff, bool noEr
             playerName = () => Main.AllPlayerNames[playerId];   
         }
 
-        _replaceDictionary["PlayerName"] = playerName;
+        //if (GameStates.IsNormalGame)
+            _replaceDictionaryNormalOptions["PlayerName"] = playerName;
+        //else if (GameStates.IsHideNSeek)
+            _replaceDictionaryHideNSeekOptions["PlayerName"] = playerName;
+
```

Comment:
For some reason, when creating a lobby, `"PlayerName"` is null
Need to find a solution to the problem

Reply:
I found out that this only happens when creating a local lobby, I don’t know why, lol

Reply:
Fixed